src/atomate2/abinit/jobs/response.py

```diff
@@ -0,0 +1,297 @@
+
+
+@job
+def generate_dte_perts(
+    gsinput: AbinitInput,
+    # TODO: or gsinput via prev_outputs?
+    skip_permutations: bool | None = False,
+    phonon_pert: bool | None = False,
+    ixc: int | None = None,
+) -> list[dict[str, int]]:
+    """
+    Generate the perturbations for the DTE calculations.
+
+    Parameters
+    ----------
+    gsinput : an |AbinitInput| representing a ground state calculation,
+        likely the SCF performed to get the WFK.
+    skip_permutations: Since the current version of abinit always performs
+        all the permutations of the perturbations, even if only one is asked,
+        if True avoids the creation of inputs that will produce duplicated outputs.
+    phonon_pert: is True also the phonon perturbations will be considered.
+        Default False.
+    ixc: Value of ixc variable. Used to overwrite the default value read
+        from pseudos.
+    """
+    # Call Abinit to get the list of irreducible perturbations
+    gsinput = gsinput.deepcopy()
+    gsinput.pop_vars(["autoparal", "npfft"])
+    perts = gsinput.abiget_irred_dteperts(
+        phonon_pert=phonon_pert, ixc=ixc
+    )  # TODO: quid manager?
+
+    if skip_permutations:
+        perts_to_skip: list = []
+        reduced_perts = []
+        for pert in perts:
+            p = (
+                (pert.i1pert, pert.i1dir),
+                (pert.i2pert, pert.i2dir),
+                (pert.i3pert, pert.i3dir),
+            )
+            if p not in perts_to_skip:
+                reduced_perts.append(pert)
+                perts_to_skip.extend(itertools.permutations(p))
+
+        perts = reduced_perts
+
+    return perts
+
+
+@job
+def run_rf(
+    perturbations: list[dict],
+    rf_maker: ResponseMaker,
+    prev_outputs: list[str] | None = None,
+) -> Flow:
+    """
+    Run the RF calculations.
+
+    Parameters
+    ----------
+    perturbations : a list of dict with the direction of the perturbation
+        under the Abipy format.
+    rf_maker : Maker to create a job with a Response Function ABINIT calculation.
+    prev_outputs : a list of previous output directories
+    """
+    rf_jobs = []
+    outputs: dict[str, list] = {"dirs": []}
+
+    if isinstance(rf_maker, (DdeMaker, DteMaker)):
+        prev_outputs = [item for sublist in prev_outputs for item in sublist]
+
+    for ipert, pert in enumerate(perturbations):
+        rf_job = rf_maker.make(
+            perturbation=pert,
+            prev_outputs=prev_outputs,
+        )
+        rf_job.append_name(f"{ipert+1}/{len(perturbations)}")
+
+        if isinstance(rf_maker, DdeMaker):
+            rf_job = update_user_abinit_settings(rf_job, {"irdddk": 1, "ird1wf": 0})
+
+        rf_jobs.append(rf_job)
+        outputs["dirs"].append(rf_job.output.dir_name)  # TODO: determine outputs
+
+    rf_flow = Flow(rf_jobs, outputs)
+    # TODO: maybe use smth else than replace to allow run_rf and rf_maker
+    # to run with different config, see if okay with dependencies, etc
```

Comment:
After discussion it seems a replace is the only viable option... so it would be nice to fix this issue of the manager config being carried over the replaced jobs independent of dynamic=False (I tested it)